data/free worlds start.txt

```diff
@@ -1615,6 +1774,122 @@ mission "FW Pirates 4.1"
+			`	JJ jumps in before you can answer. "Only Tomek said anything about conquering. Captain <last> saw the futility of annexing Greenrock as soon as he saw the planet up close. He never made any claim to the planet. He simply demanded they give up their slaves and get out of his way as he destroyed their piracy infrastructure. The recording I provided to this committee proves this."`
+			`	The older senator presses on. "Yet still, Captain <last> attacked that pirate base. He disobeyed Senate orders, he disobeyed the Council's orders, and now this recording proves that he even disobeyed the rogue Council member who ordered the illegal strike? He took an oath of loyalty, and his actions show no loyalty to any proper authority."`
+			`	JJ stands up and paces in between you and the senators. "Captain <last> became a starship captain because he yearned for freedom. He joined The Free Worlds because he believes in freedom. He has risked his life in defense of that freedom many times, and he intends to go on risking his life for The Free Worlds. He attacked Greenrock to free us from its constant piracy. He freed its slaves. He left Greenrock free behind him. Senator, What higher authority is there than the cause of freedom? If you cannot respect what Captain <last> done, then the Senate of The Free Worlds is unworthy of its name."
+			`	A confused roar breaks out. A number of senators cry out their disapproval, but an equal number of senators thump their armrests or clap in approval. The older senator purses his lips, rubs his chin, and furrows his brow. Ariana scowls. JJ scowls back at her, sits down, and leans over to you. "Now we'll see how much backbone the senate truly has."
+				goto sentence
+			label hero
+			`	"Indeed," says the senator. "And I bet it wasn't long after the battle that you realized saving the galaxy isn't that easy."`
+			`	JJ shakes his head as he leans forward to respond. "Senator, this was never about being a hero to the galaxy to Captain <last>. This was about being a hero to those slaves. Many of them have been waiting for generations to escape to a chance at a better life."`
+			`	The old senator blinks. "And you think Captain <last> can provide them a better life?"`
+			`	JJ shrugs. "They're free to find it themselves, just as we are. The free slaves seem determined to become their own heroes from now on, but I would think that the Senate and Council of the Free Worlds should be happy to help them find their way if ever they ask."`
+			label sentence
+			`	A third senator speaks up, a woman whose name card identifies her as Sara, from Skymoot. "That will be all, Captain <last>. The Senate will confer and decide whether we want your continued services."`
+			`	You are led back out to the lobby, where you wait for hours. Finally, JJ comes and leads you to  Senator Sara's office. `
```

Comment:
Bladewood had me thinking that you were pointing out a grammar issue. "Senator Sara" is identified as such in the original version of the mission, which is a little weird, because I think if "Sara" as a first name.